src/lib/ext-activitylog.js

```diff
@@ -176,6 +182,23 @@ class Controller {
   handleNewLogs(logs) {
     this.model.addNewLogs(logs);
     this.view.addTableRows(logs);
+    this.updateFilterCheckboxes(logs);
+  }
+
+  updateFilterCheckboxes(logs) {
+    const apiNameLogs = logs.filter((log) => {
+      if (log.type !== 'content_script') {
+        return true;
+      } else {
+        //log.name contains content script url
```

Comment:
It's good that you put a comment here, because otherwise the logic is not as obvious.

When you remove the `.contentScriptUrls` line  (as I suggested elsewhere), please keep the comment, so that future readers know why content script logs are filtered.

```
// log.name contains the script URL instead of the API name for content scripts.
```